.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+**JIRA Ticket:** [XXX-###](https://mfslcapstone.atlassian.net/browse/XXX-###)
+
```

Comment:
should add comment that branch names and pr names should include the ticket number